kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeAppPanel.js

```diff
@@ -462,11 +464,11 @@ define([
             var $helpHeader = $('<div>')
                 .append(
                     $('<h1>')
-                    .css({
-                        display: 'inline',
-                        'padding-right': '8px'
-                    })
-                    .append(this.help.$helpTitle))
+                        .css({
+                            display: 'inline',
+                            'padding-right': '8px'
+                        })
+                        .append(this.help.$helpTitle))
                 .append(this.help.$helpVersion);
 
             this.help.$helpBody = $('<div>')
```

Comment:
Missing semicolon.

Reply:
where? also pretty sure I didn't touch this section, not sure why its green

Reply:
Looks like it got tweaked by your editor's linter.
... and it's line 474. Which I don't think you touched, either. But hey, now's the chance to make it better! :)